src/Psy/ExecutionLoop/ForkingLoop.php

```diff
@@ -166,6 +166,8 @@ private function serializeReturn(array $return)
                 $serializable[$key] = $value;
             } catch (\Exception $e) {
                 // we'll just ignore this one...
+            } catch (\Error $e) {
```

Comment:
The regular loop does different things with `Error` vs `TypeError` vs `Exception`, so let's leave it there.